scipy/stats/_morestats.py

```diff
@@ -1133,13 +1133,19 @@ def _boxcox_inv_lmbda(x, y):
     return np.real(-num / np.log(x) - 1 / y)
 
 
-def boxcox_normmax(x, brack=None, method='pearsonr', optimizer=None):
+def boxcox_normmax(
+    x, ymax=None, brack=None, method='pearsonr', optimizer=None
```

Comment:
This breaks backwards compatibility
```suggestion
    x, brack=None, method='pearsonr', optimizer=None, *, ymax=None
```